.github/DISCUSSION_TEMPLATE/help.yml

```
@@ -0,0 +1,25 @@
+---
+body:
+  - type: textarea
+    attributes:
+      label: Summary
+      description: How do you need help?
+      placeholder: I need help with...
+      validations:
+        required: true
+  - type: textarea
+    attributes:
+      label: Additional information
+      description: Share your Storybook configuration (`main.js or ts`), any error messages, or relevant dependencies
+      render: js
+      validations:
+        required: false
+  - type: input
+    attributes:
+      label: Share an example
+      description: Help us debug your issue by creating a minimal reproduction. You can do this with StackBlitz by heading to https://storybook.new!
```

Because of the exclamation mark, this wasn't a valid link.

```suggestion
      description: Help us debug your issue by creating a minimal reproduction. You can do this with StackBlitz by heading to [https://storybook.new](https://storybook.new)!
```

---

tvym kyle!